eis_toolkit/exploratory_analyses/statistical_tests.py

```diff
@@ -52,27 +53,64 @@ def chi_square_test(data: pd.DataFrame, target_column: str, columns: Optional[Se
 
 
 @beartype
-def normality_test(data: pd.DataFrame) -> dict:
+def normality_test(
+    data: Union[pd.DataFrame, np.ndarray], columns: Optional[Sequence[str]] = None
+) -> Union[dict, Tuple]:
     """Compute Shapiro-Wilk test for normality on the input data.
 
-    It is assumed that the input data is normally distributed and numeric, i.e. integers or floats.
-
     Args:
-        data: Dataframe containing the input data.
+        data: Dataframe or numpy array containing the input data.
+        columns: Optional Columns to be used for testing.
 
     Returns:
         Test statistics for each variable.
```

Comment:
Describe here that the output format depends on the input format (Numpy array / DF) in some way.